packages/example-getting-started/package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "@loopback/context": "^4.0.0-alpha.29",
     "@loopback/core": "^4.0.0-alpha.31",
+    "@loopback/openapi-spec": "^4.0.0-alpha.23",
```

Comment:
Why is this being added? (Don't see it being used in the package).

Reply:
It seems to be used in `todo.model.ts`, but the compiler didn't complain about the package not being a dependency though, it's weird. It's entirely unrelated to migrating to `RestApplication` btw.

Reply:
I think it's because this example still uses top-down approach to define the `schema` for a model.

Reply:
It's weird indeed that it wasn't complaining about the dependency being missing before. Makes me wonder if something is broken / needs to be investigated with out build process. (Not a blocker for this PR ~ Great catch!)